src/schema/v2/homeView/experiments/HomeViewExperiments.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  GraphQLObjectType,
+  GraphQLNonNull,
+  GraphQLString,
+  GraphQLFieldConfig,
+  GraphQLList,
+  GraphQLInt,
+} from "graphql"
+import {
+  FeatureFlag,
+  getExperimentVariant,
+  getFeatureFlag,
+} from "lib/featureFlags"
+import { ResolverContext } from "types/graphql"
+import { CURRENTLY_RUNNING_EXPERIMENTS } from "./experiments"
+
+const HomeViewExperiment = new GraphQLObjectType<any, ResolverContext>({
+  name: "HomeViewExperiment",
```

Comment:
This seems like a generic-y schema to get currently running experiments _in general_. Should this be moved out of `homeView` and made a more generic type? (and then can have a specific homeview field/resolver).

Reply:
Yep, same thought here! 

I hadn't done it yet because I was staring side-eyed at this similar type… 

https://github.com/artsy/metaphysics/blob/c675d49b4c8ebafa2d2784b0db286cc32df4bbee/src/schema/v2/admin/featureFlags.ts#L17-L18

…which has a substantially different shape, I think because it's based on a different Unleash endpoint / has a different purpose. So it may make sense to keep the new type added in this PR, but move it to a common location, just as you are suggesting.

Reply:
Could we make this type independent from the home view and more generic? I guess, potentially it could be used in other places as well (`HomeViewExperiment` -> `Experiment`, or `userExperiment`, or something similar).

Reply:
I moved `HomeViewExperiment` to the more general `UnleashFeatureFlag` (named as such because it applies to `release` flags as well as `experiment` flags) in 3995f2df761c3e7d24d5d9b9b48d8fc6a56e28f0